test/common.jl

```diff
@@ -1,7 +1,7 @@
 
 gn_global_keys = Set(["per_unit","undefined_key"])
 
-generic_network_data = JSON.parse("""{
+generic_network_data = JSON.parse("""{"it": {"foo": {
```

Comment:
Just to make sure we do not have any regressions. I would recommend keeping this `generic_network_data` as is and adding a `generic_mi_network_data` variant, that is used for testing the more advance MI features.  What do you think?

Reply:
I agree that this is a good idea. I will add another network data dictionary and add extra tests.